client/verta/tests/test_versioning/test_dataset.py

```diff
@@ -165,6 +170,72 @@ def test_repr(self):
 
         assert dataset_ver.__repr__()
 
+    def test_mngd_ver_file(self, commit, in_tempdir):
+        filename = "tiny1.bin"
+        s3_key = "s3://verta-versioned-bucket/tiny-files/{}".format(filename)
+        blob_path = "data"
+
+        dataset = verta.dataset.S3(s3_key, enable_mdb_versioning=True)
+        commit.update(blob_path, dataset)
+        commit.save("Version data.")
+        dataset = commit.get(blob_path)
+
+        # download to implicit path
+        filepath = dataset.download(s3_key)
+        assert os.path.isfile(filepath)
+        assert filepath == os.path.abspath(filename)
+
+        # download to implicit path without collision
+        filepath2 = dataset.download(s3_key)
+        assert os.path.isfile(filepath2)
+        assert filepath2 != filepath
+
+        # download to explicit path with overwrite
+        last_updated = os.path.getmtime(filepath)
+        filepath3 = dataset.download(s3_key, filepath)
+        assert filepath3 == filepath
+        assert os.path.getmtime(filepath) > last_updated
+
+    def test_mngd_ver_folder(self, commit, in_tempdir):
```

Comment:
Check the content matches? It could have just created the path. Same all across.

Reply:
Will do